### PR TITLE
RPC cookie

### PR DESCRIPTION
**Pull request by pciavald**
### This pull request implements checking if a daemon is already running on the machine before launching the embedded daemon.
- created a new version of `modules/rpc/rpc.js` as `modules/rpc/rpcCookie.js` that does not subscribe to any IPC stuff, just returns cookie credentials.
   - [ ] Both files should be merged and reorganized as one module which exports 
        - a function that just returns the RPC credentials
        - function that connects IPC
- rewrote `startDaemon` function in `modules/clientBinaries/clientBinaries.js`

Still a work in progress as I've spotted a bug but didn't understand it yet. To reproduce :
```
npm run start:electron
ps aux | grep particld
```
Partgui launches but **RPC connection doesn't seem to work**.
Quit Partgui, launch `particld` the exact same way it was launched before (copy paste the output of `ps`), and run `npm run start:electron` again.
Now the embedded daemon is not being launched, because another daemon was detected by `particl-cli`'s `getinfo` command.
However, RPC connection seems to work this time, but **sync doesn't happen**.

### To sum up there are 2 remaining problems : 
- [ ] self-launching the daemon : RPC doesn't connect
- [ ] manually launched daemon : RPC connect but daemon doesn't sync

### CODE REVIEW

**Comment by rynomster** [comment]
This still has allot of work left.. Rather do a PR for the whole RPC task with arguments, etc
